src/AspNetCoreModuleV2/RequestHandlerLib/PipeOutputManager.cpp

```diff
@@ -49,6 +57,7 @@ PipeOutputManager::StopOutputRedirection()
             // if the thread is still running, we need kill it first before exit to avoid AV
             if (GetExitCodeThread(m_hErrThread, &dwThreadStatus) != 0 && dwThreadStatus == STILL_ACTIVE)
```

Comment:
wrap all calls here into logging macroses

Reply:
You referring to LOG_IF_FAILED()?

Reply:
LOG_LAST_ERROR_IF

Reply:
For things that return bool and set LastError